onboarding/source/amb_light_service.c

```diff
@@ -15,25 +22,101 @@
  */
 static void lightServiceTask(void * pvParameters);
 
+
+void adcStartConversion_selChn(adcBASE_t *adc, unsigned channel, unsigned fifo_size, unsigned group)
+{
+    /** - Setup FiFo size */
+    adc->GxINTCR[group] = fifo_size;
+    /** - Start Conversion */
+    adc->GxSEL[group] = 1 << channel;
+}
+
+void adcGetSingleData(adcBASE_t *adc, unsigned group, adcData_t *data) {
+    unsigned  buf;
+    adcData_t *ptr = data; 
+
+    /** -  Get conversion data and channel/pin id */
+    buf        = adc->GxBUF[group].BUF0;
+    ptr->value = (unsigned short)(buf & 0xFFFU);
+    ptr->id    = (unsigned short)((buf >> 16U) & 0x1FU); // int to unsigned short
+
+    adc->GxINTFLG[group] = 9U;
+
+    /**   @note The function canInit has to be called before this function can be used.\n
+    *           The user is responsible to initialize the message box.
+    */
+}
+
+uint32 getLightSensorData(void) {
```

Comment:
We typically use `uint32_t`, so I'd use the same here to stay consistent. It's the exact same as `uint32`